diopter/compiler.py

```diff
@@ -1370,20 +1388,28 @@ def __compilation_setting_parser() -> argparse.ArgumentParser:
     parser.add_argument("-o")
     parser.add_argument("-c", action="store_true")
     parser.add_argument("-S", action="store_true")
+    parser.add_argument("-MT", type=str, action="append")
+    parser.add_argument("-MQ", type=str, action="append")
```

Comment:
I guess the reason for these is to fix the deps when running the build system

Reply:
Yeah they both accept .o files as outputs so we have to make sure when we select for object files we don't consider those

Reply:
Ah right, makes sense.